libs/hdf-converters/src/sarif-mapper.ts

```diff
@@ -122,25 +118,52 @@ export class SarifMapper extends BaseConverter {
               }
             },
             impact: {path: 'level', transformer: impactMapping},
-            code: '',
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string => {
+                return JSON.stringify(vulnerability, null, 2);
+              }
+            },
             results: [
               {
                 status: ExecJSON.ControlResultStatus.Failed,
                 code_desc: {
                   path: 'locations[0].physicalLocation',
                   transformer: formatCodeDesc
                 },
-                run_time: 0,
+
                 start_time: ''
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
```

Comment:
Passthrough should likely go at the top with the rest of the header data, same applies to the other PRs.

Reply:
Since passthrough is essentially our misc. data, wouldn't placing it at the top distract away from our main focus of the files (e.g., controls & their results?). I'm of the opinion that placing it alongside the headers would make it less human readable, especially if the user just wants to find vulnerabilities etc. and don't care about whatever extraneous information we put in passthrough.